lib/task-data/tasks/boot-livecd.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Boot LiveCD',
+    injectableName: 'Task.Os.Boot.LiveCD',
+    implementsTask: 'Task.Base.Os.Install',
+    options: {
+        profile: 'boot-livecd.ipxe',
+        completionUri: 'renasar-ansible.pub',
```

Comment:
Not sure what you are talking about... The booted OS queries the completionUri to notify the boot has completed.  Why would IPXE reference it?